src/codeu/chat/client/commandline/Panel.java

```diff
@@ -30,7 +33,46 @@
     void invoke(List<String> args);
   }
 
+  class TimerCommand {
```

Comment:
No, I mean you can define a new interface `Action` for the new class `Scheduled`. These names are just suggestions; if you have a better name in mind, go with that.